### PR TITLE
Gives more specific direction for adding images to email

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by lamagnifica** [comment]
## [DOC-2830](https://openedx.atlassian.net/browse/DOC-2830)

Adds more explicit directions about using the web URL to include an image in email.

### Reviewers
- [x] Subject matter expert: @jaakana
- [x] Doc team review (sanity check): @pdesjardins 

### Testing
- [x] Ran ./run_tests.sh without warnings or errors

### Post-review
- [ ] Add description to release notes task as a comment
- [ ] Squash commits

File: en_us/shared/course_assets/course_files.rst [hunk 1]
```diff
@@ -82,19 +82,23 @@ that is assigned to it. On the **Files & Uploads** page, the **URL** column
...
-  an HTML Component`.
+  you use the **Studio** URL.
+
+  For more information, see :ref:`Add an Image to an HTML Component`.
 
 * To provide a link to the file or image from outside the course, including in
   the body of an email message that you send to course participants from the
-  LMS, you use the **Web** URL. For more information about sending email
-  messages, see :ref:`Send_Bulk_Email`.
+  LMS, you use the **Web** URL. Do not use the ``/static/`` URL in an email
```
Comment (pdesjardins) [inline en_us/shared/course_assets/course_files.rst]: "Do not use" is imperative, which we reserve for procedures, right? The sentence before is structured so it isn't imperative.

I don't really think you need to do anything with this, I just felt compelled to point it out.

File: en_us/shared/manage_live_course/bulk_email.rst [hunk 2]
```diff
@@ -49,10 +49,61 @@ canceled** after they are sent. If you plan to send a message to all course
 participants, be sure to review each draft carefully, and send the message to
 yourself first for thorough testing.
 
+.. _Options for Email Message Text:
+
+*******************************
+Options for Email Message Text
```
Comment (pdesjardins) [inline en_us/shared/manage_live_course/bulk_email.rst]: Composing Email Message Text? Just for the gerundization?

File: en_us/shared/manage_live_course/bulk_email.rst [hunk 3]
```diff
@@ -49,10 +49,61 @@ canceled** after they are sent. If you plan to send a message to all course
...
+*******************************
+
+When you compose an email message, you can take advantage of the following
```
Comment (pdesjardins) [inline en_us/shared/manage_live_course/bulk_email.rst]: you can use the following

File: en_us/shared/manage_live_course/bulk_email.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/shared/manage_live_course/bulk_email.rst]: that's what I started with!!!

File: en_us/shared/manage_live_course/bulk_email.rst (hunk 3 above)
Comment (lamagnifica) [inline en_us/shared/manage_live_course/bulk_email.rst]: yes

File: en_us/shared/manage_live_course/bulk_email.rst [hunk 4]
```diff
@@ -49,10 +49,61 @@ canceled** after they are sent. If you plan to send a message to all course
...
+Images
+======
+
+Messages can include images. To include an image, you must first add the file
+to your course on the **Files & Uploads** page. For an email message, you copy
+the **Web** URL that the system assigns to the image. Then, in the email
+message editor, you select the **Insert/edit image** icon to add the web URL.
+
+.. note:: To ensure that your course assets are transmitted securely, add the
+``https://`` preface to the web URL. An example follows.
```
Comment (pdesjardins) [inline en_us/shared/manage_live_course/bulk_email.rst:84]: Did you verify that this note is formatted correctly? It looks like the second line needs to be indented.

File: en_us/shared/course_assets/course_files.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/course_assets/course_files.rst]: verbatim from the PR.

File: en_us/shared/manage_live_course/bulk_email.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/shared/manage_live_course/bulk_email.rst:84]: didn't run tests yet, good catch

File: en_us/shared/manage_live_course/bulk_email.rst [hunk 5]
```diff
@@ -49,10 +49,61 @@ canceled** after they are sent. If you plan to send a message to all course
...
+
+For more information, see :ref:`Add Files to a Course`.
+
+=========
+Keywords
+=========
+
+Messages can include variables that represent values that are specific to each
+message recipient, such as learner name or user ID. These variables, called
+keywords, are substituted in the message text when a message is sent.
```
Comment (pdesjardins) [inline en_us/shared/manage_live_course/bulk_email.rst]: The LMS substitutes these variables, called keywords, with actual values when it sends a message.

Just a thought.

File: en_us/shared/manage_live_course/bulk_email.rst [hunk 6]
```diff
@@ -49,10 +49,61 @@ canceled** after they are sent. If you plan to send a message to all course
...
+
+For more information, see :ref:`Use Keywords in Messages`.
+
 .. _Send_Bulk_Email:
 
 **************************************************
-Send Email Messages to Course Participants
+Send an Email Message to Course Participants
```
Comment (pdesjardins) [inline en_us/shared/manage_live_course/bulk_email.rst:105]: Sending an Email Message to Course Participants

File: en_us/shared/manage_live_course/bulk_email.rst [hunk 7]
```diff
@@ -68,22 +119,16 @@ To send an email message to course participants, follow these steps.
...
-   HTML Components`.
+#. Enter the text for the message. Message text is required.
 
-   You can use keywords in messages, to have values that are specific to each
-   recipient substituted in the content when messages are sent. For more
-   information, see :ref:`Use Keywords in Messages`.
+   For more information about options for composing your message, see
+   :ref:`Options for Email Message Text`.
 
 #. Select **Send Email**. You can perform other tasks on the Instructor
```
Comment (pdesjardins) [inline en_us/shared/manage_live_course/bulk_email.rst:132]: Not part of your changes, I would use lower case for instructor dashboard.

**Comment by pdesjardins** [comment]
This looks good to me. :+1: 

I think I misunderstood the source though. Is this from another PR?

File: en_us/shared/manage_live_course/bulk_email.rst (hunk 5 above)
Comment (lamagnifica) [inline en_us/shared/manage_live_course/bulk_email.rst]: good, I was struggling, thanks

File: en_us/shared/manage_live_course/bulk_email.rst (hunk 6 above)
Comment (lamagnifica) [inline en_us/shared/manage_live_course/bulk_email.rst:105]: procedure, so no gerund...

**Comment by lamagnifica** [comment]
see DOC-2830 (link in description)

File: en_us/shared/manage_live_course/bulk_email.rst (hunk 7 above)
Comment (lamagnifica) [inline en_us/shared/manage_live_course/bulk_email.rst:132]: changed 2x

**Comment by lamagnifica** [comment]
@jaakana when you get a chance. Hope this is an improvement!

**Comment by JAAkana** [comment]
Thanks for this work! (:palm_tree:)

**Pull request merged**